api/odigos/v1alpha1/instrumentationconfig_types.go

```diff
@@ -24,11 +24,6 @@ type InstrumentationConfigSpec struct {
 	// true when the runtime details are invalidated and should be recalculated
 	RuntimeDetailsInvalidated bool `json:"runtimeDetailsInvalidated,omitempty"`
 
-	// config for this workload.
-	// the config is a list to allow for multiple config options and values to be applied.
-	// the list is processed in order, and the first matching config is applied.
-	Config []WorkloadInstrumentationConfig `json:"config,omitempty"`
```

Comment:
If we remove this, we should remove the struct definition for `WorkloadInstrumentationConfig`.

Reply:
right!
good catch. fixed